test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -155,6 +155,21 @@ def test_struct_payload_non_utf8_log
     end
   end
 
+  def test_tags
+    setup_gce_metadata_stubs
+    [123, 'test', 'germanß', 'chinese中'].each do |tag|
```

Comment:
Because I added that very last minute, and forgot to duplicate it. Just changed.